packages/shared/src/components/cards/v1/SharedPostCardFooter.tsx

```diff
@@ -0,0 +1,55 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { Post } from '../../../graphql/posts';
+import { cloudinary } from '../../../lib/image';
+import { IconSize } from '../../Icon';
+import { CardImage, CardVideoImage } from './Card';
+
+type SharedPostCardFooterProps = {
+  isShort: boolean;
+  isVideoType?: boolean;
+} & Pick<Post, 'sharedPost'>;
+
+export const SharedPostCardFooter = ({
+  sharedPost,
+  isShort,
+  isVideoType,
+}: SharedPostCardFooterProps): ReactElement => {
+  const ImageComponent = isVideoType ? CardVideoImage : CardImage;
+  return (
+    <div
+      className={classNames(
+        'mb-2 flex h-auto min-h-0 w-full flex-auto gap-3 rounded-12 border border-theme-divider-tertiary p-3',
+        isShort ? 'flex-row items-center' : 'flex-col  mobileL:flex-row',
+      )}
+    >
+      <div
+        className={classNames(
+          'flex flex-col mobileL:flex-1',
+          isShort ? 'line-clamp-4 w-8/12 pr-3' : 'line-clamp-2',
+        )}
+      >
+        {sharedPost.title}
+        <span className="text-theme-label-quaternary">
+          {sharedPost.summary}
```

Comment:
Just making sure, in the notes this title is said to be max 2 line, so I'm guessing this needs to always be line-clamp-2?